File: JvmClient/src/commonMain/kotlin/spaceEngineers/model/MedicalsData.kt
package spaceEngineers.model

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
data class MedicalsData(
    @SerialName("MedicalRooms")
    val medicalRooms: List<MedicalRoom>,
    @SerialName("Factions")
    val factions: List<Faction>,
    @SerialName("ShowFactions")
    val showFactions: Boolean,
    @SerialName("ShowMotD")
    val showMotD: Boolean,
    @SerialName("IsMotdOpen")
    val isMotdOpen: Boolean,
    @SerialName("Paused")
    val paused: Boolean,
    @SerialName("IsMultiplayerReady")
    val isMultiplayerReady: Boolean,
    @SerialName("RespawnButton")
    val respawnButton: GuiControlBase?,
)

File: Source/Ivxr.SpaceEngineers/WorldModel/Screen/MedicalsData.cs
using System.Collections.Generic;

namespace Iv4xr.SpaceEngineers.WorldModel.Screen
{
    public class MedicalsData
    {
        public List<MedicalRoom> MedicalRooms;
        public List<Faction> Factions;
        public bool ShowFactions;
        public bool ShowMotD;
        public bool IsMotdOpen;
        public bool Paused;
        public bool IsMultiplayerReady;
        public GuiControlBase RespawnButton;
    }
}
